app/api/access_codes.py

```diff
@@ -160,9 +161,8 @@ def before_get(self, args, kwargs):
                 raise ObjectNotFound({'parameter': '{id}'}, "Access Code:  not found")
 
             if not has_access('is_coorganizer', event_id=access.event_id):
-                raise UnprocessableEntityError(
-                    {'source': ''}, "Please verify your permission"
-                )
+                raise UnprocessableEntityError({'parameter': 'id'},
```

Comment:
Remove paramater id, just second argument is enough

Reply:
Here only `'source': 'id'` will be fine!!